src/components/pages/units-page/edit-unit-dialog.tsx

```diff
@@ -317,19 +348,30 @@ export const EditUnitDialog = (props: EditUnitDialogProps) => {
           <Grid item xs={12}>
             <Typography className={classes.headerLabel}>Muster Configuration</Typography>
             <div>
-              <FormControlLabel
-                control={(
-                  <Checkbox
-                    checked={overrideDefault}
-                    color="primary"
-                    disabled={formDisabled}
-                    onChange={() => setOverrideDefault(!overrideDefault)}
-                  />
-                  )}
-                label="Override Default Group Muster Requirements"
-              />
+              <ToggleButtonGroup
+                value={musterConfigMode}
+                exclusive
+                onChange={(_, value) => {
+                  if (value !== null) {
+                    setMusterConfigMode(value);
+                  }
+                }}
+                aria-label="muster configuration mode"
+              >
+                <ToggleButton value={MusterConfigMode.Default}>Default</ToggleButton>
+                <ToggleButton value={MusterConfigMode.Custom}>Custom</ToggleButton>
+                <ToggleButton value={MusterConfigMode.None}>None</ToggleButton>
+              </ToggleButtonGroup>
             </div>
-            {musterConfiguration.length > 0 && (
+            {showNoneMessage && (
+              <p>
+                This unit is not required to muster.
+                {musterConfigMode === MusterConfigMode.Default && (
+                  <em> * using default muster requirements</em>
```

Comment:
Very minor, but the asterisk looks a little awkward to me in the UI on the same line. I think it might look better as `(using default muster requirements)`.